pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -95,6 +95,7 @@ public void initMocks() throws Exception {
     when(mockRunnerBuilder.webSocketConfiguration(any())).thenReturn(mockRunnerBuilder);
     when(mockRunnerBuilder.dataDir(any())).thenReturn(mockRunnerBuilder);
     when(mockRunnerBuilder.bannedNodeIds(any())).thenReturn(mockRunnerBuilder);
+    when(mockRunnerBuilder.metricsSystem(any())).thenReturn(mockRunnerBuilder);
```

Comment:
This was a whole lot more lines to add a new object to the builder before the builder remodel.